### DISCUSSION TITLE
Brightsign Script + Plugin

### DISCUSSION THREAD

**Discussion by thefennecdeer** [museumsvictoria/nodel#308] [General]
Heya guys!
I've created a Brightsign plugin and accompanying Nodel script thats plug and play with **any new or existing presentation** in BA:Connected.

The previous control scripts I've seen are mutually exclusive with BA:Connected, in that you use one or the other. This plugin adds status reports, playback control, volume and mute control, and power save control to a BA:Connected presentation, with all its logic and customisability!

Its available here: [https://github.com/thefennecdeer/Nodel-Brightsign](https://github.com/thefennecdeer/Nodel-Brightsign/)

Let me know if you have any issues!

**Comment by scroix** [comment]
There is a really old Brightsign [recipe](https://github.com/museumsvictoria/nodel-recipes/blob/master/Brightsign/script.py) which this would be a fantastic replacement for. Would you consider submitting to the public recipes repo?

**Comment by justparking** [comment]
If you haven't spotted it, I've create a [PR](https://github.com/museumsvictoria/nodel-recipes/pull/81) which shows a recipe I've used for monitoring only.

I think it overlaps _slightly_ with yours, @thefennecdeer.

Feel free to cherry pick anything that'd be useful, delete the branch and close the PR.
